buildtools/gradle/testdata/tasks-output.txt

```diff
@@ -0,0 +1,77 @@
+------------------------------------------------------------
```

Comment:
It might be helpful to cut out some lines in this file so that its easier to read. I try to focus on trying to only have as many cases that could trip up the parsing logic. This could also be applied to `complex.txt`.